int32 personId
---
int32 status
int32 recognitionId

#recognition status constansts
int32 REGISTRATION_SUCCESSFULL=0
int32 REGISTRATION_FAILED=1
int32 REGISTRATION_FAILED_ALREADY_REGISTERED=2
int32 REGISTRATION_FAILED_FACE_NOT_DETECTED=3
int32 REGISTRATION_FAILED_FACE_NOT_CLEAR=4
int32 REGISTRATION_FAILED_PERSON_TO_FAR=5
int32 REGISTRATION_FAILED_PERSON_TO_CLOSE=6
